src/Stripe.net/Services/Charges/StripeChargeLevel3Options.cs

```diff
@@ -5,24 +5,22 @@ namespace Stripe
 
     public class StripeChargeLevel3Options : INestedOptions
     {
-        [JsonProperty("level3[customer_reference]")]
+        [JsonProperty("customer_reference")]
         public string CustomerReference { get; set; }
 
-        // this will actually send `line_items`. this is to flag it for the middleware
-        // to process it as a plugin
-        [JsonProperty("level3[line_items]")]
+        [JsonProperty("line_items")]
```

Comment:
This one is the kind of things we should test since we removed the plugin. We don't have a test for that property today I think

Reply:
Sure, but now that we've got rid of plugins, things should be a lot more consistent. We should exhaustively test the new encoding logic (there's already decent coverage, but we can add more) and that should give us enough confidence that we don't need to test each specific instance in the API.